docs/control-structures.rst

```diff
@@ -60,31 +63,73 @@ Input parameters and output parameters can be used as expressions in
 the function body.  There, they are also usable in the left-hand side
 of assignment.
 
+
+.. _multi-return:
+
+Returning Multiple Values
+-------------------------
+
+When a function has multiple output parameters, ``return (v0, v1, ...,
+vn)`` can return multiple values.  The number of components must be
+the same as the number of output parameters.
+
+
+.. index:: return array, return string, array, string, array of strings, dynamic array, variable sized array
+
+.. _return-array:
+
+Returning Strings and Arrays
+----------------------------
+
+You can return strings and arrays from Solidity functions, see `array_receiver_and_returner.sol <https://github.com/fivedogit/solidity-baby-steps/blob/master/contracts/60_array_receiver_and_returner.sol>`_ for an example.
+
+What is problematic, though, is returning any variably-sized data (e.g. a variably-sized array like ``uint[]`` or an array of strings ``string[]``) from a function **called from within Solidity**. This is a limitation of the EVM and will be solved with the next protocol update.
```

Comment:
This is not an issue anymore.

Reply:
@chriseth I might just leave the example line, or maybe add our own example below, mostly so this is in the index and search results if people are looking for it.